dist/webgl/program.js

```diff
@@ -244,8 +244,8 @@ function getUniformSetter(gl, glProgram, info, isArray) {
 
   var matrix = false;
   var vector = true;
-  var glFunction = undefined;
-  var TypedArray = undefined;
+  var glFunction = void 0;
```

Comment:
Interesting. I wonder why these changed? Did you just pull in a later version of babel?